lib/plugin/instance.rb

```diff
@@ -1476,4 +1489,8 @@ def find_locale_file(locale_chain, path)
   def register_permitted_bulk_action_parameter(name)
     DiscoursePluginRegistry.register_permitted_bulk_action_parameter(name, self)
   end
+
+  def default_admin_route
+    { label: "#{name.underscore}.title", location: name, use_new_show_route: true }
```

Comment:
I think we should still support the frontend translation fallback for the category names:

https://github.com/discourse/discourse/blob/78a857931c1bfbc3d309ec6706fdfd3a612b68df/app/assets/javascripts/admin/addon/models/admin-plugin.js#L51-L76

Just for the cases where we have missed adding a title to some plugins. We currently do this in the AdminPlugin model:

https://github.com/discourse/discourse/blob/78a857931c1bfbc3d309ec6706fdfd3a612b68df/app/assets/javascripts/admin/addon/models/admin-plugin.js#L51-L76

Otherwise we would have to use all-the-plugins to find the ones with a missing title and add it

Reply:
Oh, yes. I realized I even suggested the same thing earlier. 😅